src/state/cache/thread-mutes.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+
+type StateContext = Record<string, boolean>
+type SetStateContext = (uri: string, value: boolean) => void
+
+const stateContext = React.createContext<StateContext>({})
+const setStateContext = React.createContext<SetStateContext>(
+  (_: string) => false,
+)
+
+export function Provider({children}: React.PropsWithChildren<{}>) {
+  const [state, setState] = React.useState<StateContext>({})
```

Comment:
Any objections to using a Map? With lazy init, `useState(() => new Map())`. Generally Maps are better than objects when your keys are arbitrary.